website/content/docs/connect/config-entries/partition-exports.mdx

```diff
@@ -0,0 +1,207 @@
+| `Partition` | String value that specifies the name of the partition that contains the services you want to export. | Required | None |
+| `Services` | List of objects that specify which services to export. See [`Services`](#services) for details. | Required | None| 
+| `Meta` | Object that defines a map of the max 64 key/value pairs. | Optional | None |
+
+#### `Services`
+
+The `Services` parameter contains one or more lists of parameters that specify which services to export, which namespaces the services reside, and the destination partition for the exported services. Each list in the `Services` block must contain the following parameters:
+
+* `Name`: Specifies the name of the service to export. You can use a asterisk wildcard (`*`) to include all services in the namespace. 
+* `Namespace`: Specifies the name space containing the services to export. You can use a asterisk wildcard (`*`) to include all namespaces in the partition.
+* `Consumers`: Specifies one ore more objects that identify a destination partition for the exported services. 
+
+## Example
+
+The following example configures the agent to export the `billing` service from the `default` namespace of the `finance` admin partition to the `frontend` and `backend` partitions. Additionally, all services in all namespaces within the `finance` partition will be exported to the `monitoring` partition. 
+
+<CodeTabs heading="" tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+<CodeBlockConfig>
+
+```hcl
+Kind = "partition-exports" 
+Partition = "finance"
+
+Services = [
+  {
+    Name = "billing"
+    Namespace = "default"
+    Consumers = [
+        {
+            Partition  = "frontend"
+        },
+        {
+            Partition  = "backend"
+        }
+    ]
+  },
+  {
+    Name      = "*"
+    Namespace = "*"
+    Consumers = [
+        {
+            Partition  = "monitoring"
+        }
+    ]
+  }
+]
+```
+
+</CodeBlockConfig>
+<CodeBlockConfig>
+
+```yaml
+Kind: partition-exports
+Partition: finance
+Services:
+- Consumers:
+  - Partition: frontend
+  - Partition: backend
+  Name: billing
+  Namespace: default
+- Consumers:
+  - Partition: monitoring
+  Name: '*'
+  Namespace: '*'
+```
+
+</CodeBlockConfig>
+<CodeBlockConfig>
+
+```json
+"Kind": "partition-exports",
+  "Partition": "finance",
+  "Services": [
+    {
+      "Consumers": [
+        {
+          "Partition": "frontend"
+        },
+        {
+          "Partition": "backend"
+        }
+      ],
+      "Name": "billing",
+      "Namespace": "default"
+    },
+    {
+      "Consumers": [
+        {
+          "Partition": "monitoring"
+        }
+      ],
+      "Name": "*",
+      "Namespace": "*"
+    }
+  ]
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+## Reading Services
+
+When an exported service has been imported to another partition, you can use the `health` REST API endpoint to query the service on the consumer partition. The following example queries the `finance` partition for the imported `billing` service:
```

Comment:
A caveat here is that they request will need an ACL token with `service:write` permissions within their partition. So say that the request below is from service "web" in the "frontend" partition, the request will need a token with write permissions to "web" in "frontend". Though the export is available to all services in the consumer partition, so any service write permissions in "frontend" will work.